orbit/utils/simulation.py

```diff
@@ -5,14 +5,13 @@
 from orbit.exceptions import IllegalArgument
 
 
-def make_trend(series_len, rw_loc=0.001, rw_scale=0.1, type='rw', seed=1):
+def make_trend(series_len, rw_loc=0.0, rw_scale=0.1, method='rw', seed=1):
     """ Module to generate time-series trend with different methods
-
     Parameters
     ----------
     series_len: int
         total length of series
-    type: str ['arma', 'rw']
+    method: str ['arma', 'rw']
     rw_loc: float
         mean of random walk
     rw_scale: float
```

Comment:
can you add a descriptive definition for this arg

Reply:
added